src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -626,6 +626,8 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 			livingEntity.transparentModelCondition = new Procedure("condition1");
 			livingEntity.isShakingCondition = new Procedure("condition2");
 			livingEntity.solidBoundingBox = new LogicProcedure(_true ? "condition3" : null, _true);
+			livingEntity.visualScale = new NumberProcedure(emptyLists ? null : "number1", 8);

```

Comment:
Use different numbers for different procedures in tests, so number2, 3, ..., check what number providers are already used in living entity and add more if needed

Reply:
Will get those dones on the weekend, lights when off and they go off for days at a time

Reply:
Not rushing you, when you have time ;) I just got time to review your PRs this evening :)

Reply:
Was finally able to update this + the other prs 😄